### ISSUE TITLE
[Help] How to use env var like `import.meta.env.VITE_KEY` in main/preload ?

### ISSUE THREAD

**Issue by dongivan** [electron-vite/electron-vite-vue#209]
#### Describe the problem you confuse

Vite supports env vars prefixed with "VITE_" if they are placed in the file `.env`.
However in `electron-vite-vue` template, env vars like `import.meta.env.VITE_KEY` can only be used in renderer process.

How to reproduce:
1. `npm create electron-vite` & `npm i`
2. Add `.env` file at app root with the content:
```
VITE_TEST="test string"
```
3. Modify `electron/electron-env.d.ts`:
```ts
/// <reference types="vite-plugin-electron/electron-env" />

interface ImportMetaEnv {
  readonly VITE_TEST: string
}

interface ImportMeta {
  readonly env: ImportMetaEnv
}
```
4.  Add this line into `electron/main/index.ts`:
```ts
console.log("in main process", import.meta.env.VITE_TEST)
```
5. Add this line into `electron/preload/index.ts`:
```ts
console.log("in preload", import.meta.env.VITE_TEST)
```
6. Add this line into `src/main.ts`:
```ts
console.log("in renderer process", import.meta.env.VITE_TEST)
```
7. `npm run dev`
8. Result:
<img width="876" alt="main" src="https://user-images.githubusercontent.com/6535256/180638311-9133192f-2be6-4ec3-8bbc-260a19ce565d.png">
<img width="299" alt="renderer" src="https://user-images.githubusercontent.com/6535256/180638334-1b5095ec-30c1-4d83-b799-d2dc8bdc792b.png">

**Comment by dongivan** [comment]
BTW, currently I use `dotenv` to load env vars of `.env` file into `process.env`.

**Comment by caoxiemeihao** [comment]
It's probable a conservative desing, not sure if it's rihg.
However, usually in Node.js environments, `process.env` is always avaizble.

```js
  const conf: InlineConfig = {
    // 🚧 Avoid recursive build caused by load config file
    configFile: false,
    envFile: false, // 👈👈👈👈👈👈👈
    publicDir: false,

    build: {
      emptyOutDir: false,
      minify: process.env./* from mode option */NODE_ENV === 'production',
    },
  }
```

https://github.com/electron-vite/vite-plugin-electron/blob/2df429e643192a0a9a0702b1c3b36544460950b0/packages/electron/src/config.ts#L32

---

Perhaps, this feature is avaiable in `v0.8.2`, what do you think about it. My bro 😄 @PAXANDDOS

**Comment by dongivan** [comment]
Thank you for replying!

**Comment by PAXANDDOS** [comment]
> Perhaps, this feature is available in `v0.8.2`, what do you think about it. My bro 😄 @PAXANDDOS

Honestly, I thought the env file was exposed everywhere 😅

I was thinking that we don't need `.env` files here at all, we're building desktop applications with Electron anyways. But then I remembered some cases where you need to have, for example, a hash string for modules, and since all application logic is stored in the main process, exposing environment files to it would be a great idea

@caoxiemeihao So I want to say that yes, it's better to enable the `envFile` option
